samples/matter/lock/README.rst

```diff
@@ -481,37 +548,10 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial of the Matter documentation.
 
-.. _matter_lock_scheduled_timed_access:
-
-Scheduled timed access
-======================
-
-The scheduled timed access feature is an optional Matter lock feature that can be applicable to all available lock users.
-You can use the scheduled timed access feature to allow guest users of the home to access the lock at the specific scheduled times.
-To use this feature, you need to create at least one user on the lock device, and assign credentials.
-For more information about setting user credentials, see the Saving users and credentials on door lock devices section of the :doc:`matter:chip_tool_guide` page in the :doc:`Matter documentation set <matter:index>`, and the :ref:`matter_lock_sample_remote_access_with_pin` section of this sample.
-
-You can schedule the following types of timed access:
-
-   - ``Week-day`` - Restricts access to a specified time window on certain days of the week for a specific user.
-     This schedule grants repeated access each week.
-     When the schedule is cleared, the user is granted unrestricted access.
-
-   - ``Year-day`` - Restricts access to a specified time window on a specific date window.
-     This schedule grants access only once, and does not repeat.
-     When the schedule is cleared, the user is granted unrestricted access.
-
-   - ``Holiday`` - Sets up a holiday operating mode in the lock device.
-     You can choose one of the following operating modes:
+.. _matter_lock_enabling_scheduled_timed_access:
 
-     - ``Normal`` - The lock operates normally.
-     - ``Vacation`` - Only remote operations are enabled.
-     - ``Privacy`` - All external interactions with the lock are disabled.
-       Can only be used if the lock is in the locked state.
-       Manually unlocking the lock changes the mode to ``Normal``.
-     - ``NoRemoteLockUnlock`` - All remote operations with the lock are disabled.
-     - ``Passage`` - The lock can be operated without providing a PIN.
-       This option can be used, for example, for employees during working hours.
+Enabling scheduled timed access
```

Comment:
The last 2 testing sections (Testing switching between Thread and Wi-Fi and Bluetooth LE with Nordic UART Service) also mention DK buttons

Reply:
The switching build type is currently only supported for the nRF5340 DK with the nRF7002 EK, so it doesn't need text for the nRF54 DKs.

Took out the LED mention from the UART section.